src/NinjaTools.FluentMockServer/Xunit/Attributes/MockServerCollectionAttribute.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Reflection;
+using Xunit.Sdk;
+
+namespace NinjaTools.FluentMockServer.Xunit.Attributes
+{
+    [AttributeUsage(AttributeTargets.Method)]
+    public class MockServerCollectionAttribute : BeforeAfterTestAttribute
+    {
+        public string Id { get; }
+
+        /// <inheritdoc />
+        public MockServerCollectionAttribute(string collection) : base()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this redundant 'base()' call.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4885942)